### DISCUSSION TITLE
Is the meaning of the parameter -threads in juicefs' built-in NNBench test the same as the meaning of the -reduces parameter in Hadoop's built-in NNBenchtest?

### DISCUSSION THREAD

**Discussion by ANHDY** [juicedata/juicefs#5186] [Q&A]
In the built-in NNBench test of Hadoop, there is no '-threads' parameter, but a '-reduces' parameter. However, in the built-in NNBench test of JuiceFS, there is no '-reduces' parameter. May I ask if the meaning of the '-threads' parameter setting is the same as that of the '-reduces' ? If they are not the same, is there a default value or other representation set for the ‘-reduces' parameter?

**Comment by tangyoupeng** [comment]
reduces is only used for collect map output. It did not run tests

**Reply by tangyoupeng** [reply]
threads is used in mapper to run tests
